authentication/src/AuthenticationProviderX509.cs

```diff
@@ -1,15 +1,36 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Security.Cryptography.X509Certificates;
 
 namespace Microsoft.Azure.Devices.Authentication
 {
     /// <summary>
-    /// The device authentication provider interface for X.509-based hardware security modules.
+    /// The device authentication for using an X509 certificate object.
     /// </summary>
-    public abstract class AuthenticationProviderX509 : AuthenticationProvider
+    public class AuthenticationProviderX509 : AuthenticationProvider
```

Comment:
extremely small nit: why not "AuthenticationProviderX509Certificate", since this class deals with x509 certificates.

Reply:
To me, x509 already implies "x509 certificates", so that would be a bit redundant